lib/autotune/deployer.rb

```diff
@@ -230,13 +232,49 @@ def google_doc_contents(url)
       # TODO: needs test coverage
       if needs_update
         google_client.share_with_domain(doc_key, Autotune.configuration.google_auth_domain)
-        ret = google_client.get_doc_contents(url)
+        format = format.to_sym unless format.nil?
+        ret = google_client.get_doc_contents(url, :format => format)
```

Comment:
Should this throw if it is an incompatible format? For instance if user tries to retrieve a spreadsheet as text. Right now it ignores these for spreadsheets and always handles it as an xlsx